variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/VariantClassifierTest.java

```diff
@@ -85,4 +85,72 @@ public void testHyphensInReference() {
     public void testHyphensInAlternate() {
         VariantClassifier.getVariantClassification("AA", "-", 2);
     }
+
+    //Test getVariantClassification without subSNPClass
+    @Test
+    public void testSNVsNoSnpClass() {
+        assertEquals(VariantType.SNV, VariantClassifier.getVariantClassification("A", "C"));
+    }
+
+    @Test
+    public void testDeletionsNoSnpClass() {
+        assertEquals(VariantType.DEL, VariantClassifier.getVariantClassification("TT", ""));
+    }
+
+    @Test
+    public void testInsertionsNoSnpClass() {
+        assertEquals(VariantType.INS, VariantClassifier.getVariantClassification("", "G"));
+    }
+
+    @Test
+    public void testINDELsNoSnpClass() {
+        assertEquals(VariantType.INDEL, VariantClassifier.getVariantClassification("A", "GTC"));
+        assertEquals(VariantType.INDEL, VariantClassifier.getVariantClassification("AA", "G"));
+    }
+
+    @Test
+    public void testSTRsNoSnpClass() {
+        assertEquals(VariantType.TANDEM_REPEAT, VariantClassifier.getVariantClassification("(A)5", "(A)7"));
+        assertEquals(VariantType.TANDEM_REPEAT, VariantClassifier.getVariantClassification("(G)3", "(G)7"));
+        assertEquals(VariantType.TANDEM_REPEAT, VariantClassifier.getVariantClassification("AAAAA", "AAAAAAA"));
```

Comment:
I don't think these cases should be TANDEM_REPEATS, but I could be wrong depending on the outcome of EVA-2092. I suggest leaving these cases as regular INSs or DELs for now